src/witan/workspace_peer/config.clj

```diff
@@ -1,11 +1,11 @@
 (ns witan.workspace-peer.config
   (:require [witan.workspace-peer.utils.workflow :refer :all]
             ;;
-            [witan.models.dem.ccm.fert.fertility-mvp]))
+            [witan.models.dem.ccm.fert.fertility]))
```

Comment:
it's a snapshot version, so it looks like it the namespace name just got trimmed and the snapshot got updated.